app/views/dashboard/index.html.erb

```diff
@@ -8,15 +8,39 @@
 <table class="table sortable-table" style="margin-top:50px">
   <thead>
     <tr>
-      <th>Reason</th>
+      <th>Active Reason</th>
       <th>Number caught</th>
       <th class='no-sort'>Last caught</th>
       <th>Accuracy</th>
     </tr>
   </thead>
 
   <tbody>
-    <% @reasons.each do |reason| %>
+    <% @active_reasons.each do |reason| %>
+      <tr class="<%= "active text-muted inactive-reason" if reason.inactive %>">
```

Comment:
You should be able to remove the conditional here — the reason should always be active in this block.

Reply:
Yes, I missed that.  Thanks.